### DISCUSSION TITLE
How can I enable auto launch only on Firefox?

### DISCUSSION THREAD

**Discussion by EdoanR** [wxt-dev/wxt#499] [Q&A] [answered]
Hi, I find auto launch very useful for Firefox as the extension is automatically installed as a temporary extension, but never found necessary for chrome as the extension is still there in different sessions. Is there a way to enable auto launch just for Firefox?

Currently I'm setting a `TARGET` environment in the package scripts, so I can access it on the runner config. But I'm wondering if there's a better way.

package.json
```json
"scripts": {
	"dev:firefox": "(set TARGET=firefox) && wxt -b firefox"
},
```

web-ext.config.ts
```ts
import { defineRunnerConfig } from 'wxt';

export default defineRunnerConfig({
	disabled: process.env.TARGET !== 'firefox',
});

```

**Comment by aklinker1** [comment, marked as answer]
That is the only way. You might want to install and use [`cross-env`](https://www.npmjs.com/package/cross-env) for your `dev:firefox` script:

```json
"scripts": {
  "dev:firefox": "cross-env TARGET=firefox wxt -b firefox"
},
```

But your way is fine if you're just using windows.
